modules/odc-viewing-application-topology.adoc

```diff
@@ -3,7 +3,7 @@
 
 You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you deploy an application, you are directed automatically to the *Graph view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
 
-The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List view* icon (image:odc_list_view_icon.png[title="List view icon"]) to see a list of all your applications and use the *Graph view* icon (image:odc_topology_view_icon.png[title="Topology view icon"]) to switch back to the graph view.
+The *Topology* view provides you the option to monitor your applications using the *List* view. Use the *List view* icon (image:odc_list_view_icon.png[title="List view icon"]) to see a list of all your applications and use the *Graph view* icon (image:odc_topology_view_icon.png[title="Topology view icon"]) to switch back to the graph view.
```

Comment:
I defer to you as you know your users more. But for accessibility and usability, be careful with images. They can be particularly difficult to maintain. 

Reaching out to your content strategist would help make this decision. Again, if you know to keep them, leave this content here as is :)

Reply:
@jc-berger, I agree with your point of maintenance. This is why i only use screenshots where it is actually needed for assistance. 
Here, we have used inline images to show UI icons, which might be scattered across the view. The graphics actually help readers to locate them. So, leaving the images as they are.